src/main/java/converter/JsonToXmlFormatConverter.java

```diff
@@ -0,0 +1,20 @@
+package converter;
+
+import exceptions.ConvertFormatXmlToJsonException;
+import format.FormatType;
+import format.IFormatObject;
+import format.XmlFormat;
+import org.json.JSONException;
+import org.json.XML;
+
+
+public class JsonToXmlFormatConverter implements FormatConverter {
+    @Override
+    public IFormatObject convertFormat(IFormatObject jsonFormat, FormatType formatTypeToConvert) throws ConvertFormatXmlToJsonException {
+        try {
+            return new XmlFormat(formatTypeToConvert.getXmlHeader() + XML.toString(new org.json.JSONObject(jsonFormat.getFormatDataAsString())));
```

Comment:
Initializing JSONObject as function paramter is a bad idea for production environments.